spec.html

```diff
@@ -27130,6 +27130,46 @@
       </emu-clause>
     </emu-clause>
 
+    <emu-clause id="sec-string.prototype.padstart">
+      <h1>String.prototype.padStart( maxLength [ , fillString ] )</h1>
+      <p>When the `padStart` method is called, the following steps are taken:</p>
+      <emu-alg>
+        1. Let _O_ be ? RequireObjectCoercible(*this* value).
+        1. Let _S_ be ? ToString(_O_).
+        1. Let _intMaxLength_ be ? ToLength(_maxLength_).
+        1. Let _stringLength_ be the number of elements in S.
+        1. If _intMaxLength_ is not greater than _stringLength_, return _S_.
+        1. If _fillString_ is *undefined*, let _filler_ be a string consisting solely of the code unit U+0020 (SPACE).
```

Comment:
I discussed this with @bterlson in stage 3, and this was the preferred mechanism. 

The capitalization seems to have been made consistent since I initially wrote the text - will update!